src/styles/blocks/second.scss

```diff
@@ -0,0 +1,22 @@
+.watch__second {
+  box-sizing: border-box;
+  position: absolute;
+  bottom: 198px;
+  border: 2px solid yellow;
+  background-color: yellow;
+  height: 190px;
+  transform-origin: 50% bottom;
+  animation-name: sec_rotation;
+  animation-duration: 60s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+
+  @keyframes sec_rotation {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+}
```

Comment:
[BEM] - create a separate file per each BEM block styles that have the same name as the block
it is the element of `watch` BEM block, so move it to `watch.scss`